articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+     ![Add new connection](images/azs-browser-add-connection.png)
+
+6. Click **OK**.
+
+## Create a virtual network in Public Azure
+
+You'll also need to create a virtual network in Public Azure. This virtual network will be able to send and receive traffic through the virtual network gateway. You cannot associate a virtual network with more than one gateway.
+
+1. Log in to the [public Azure portal](https://portal.azure.com).
+
+2. In the favourites panel, select **Create a resource**.
+
+   ![Create a resource option in favourites panel](images/azs-public-browser-create-resource-button.png)
+
+3. In the **New** blade, select **Networking**.
+
+   ![Networking option in New blade](images/azs-public-browser-new-networking-button.png)
+
+4. In the **Featured** section, select **Virtual network**.
+
+   ![Create new networking resource](images/azs-public-browser-new-virtual-network.png)
+
+5. In the **Create virtual network** blade, enter the following information:
+
+   - **Name** - The name of the virtual network.
+
+   - **Address Space** - The virtual network's address range in CIDR notation.
+
+   - **Subscription** - This is your Microsoft Azure subscription.
+
+   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - Select the location of the Azure resource.
+
+   - **Subnet Name** - The name of the first subnet within the virtual network.
+
+   - **Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0). It must be within the address range of the virtual network. It should be noted that the address range of a subnet currently in use cannot be edited.
+
+   - **DDoS protection** - Select the DDoS protection you would like, for more information please see the [Azure DDoS Protection Standard overview](https://docs.microsoft.com/en-gb/azure/virtual-network/ddos-protection-overview).
+
+   - **Service endpoints** - This allows you to enable one or more service endpoints for more information please see [Virtual Network Service Endpoints](https://docs.microsoft.com/en-gb/azure/virtual-network/virtual-network-service-endpoints-overview).
+
+   - **Firewall** - This allows you to enable Azure Firewall, which is a managed cloud-based network security service that protects your Azure Virtual Network resources.
+
+     ![Create new virtual network](images/azs-public-browser-new-virtual-network-blade.png)
+
+6. Click **Create**.
+
+7. After your virtual network has deployed, you can view it by clicking **All services** in the favourites panel, then selecting **Virtual networks** under the networking section.
+
+    ![Virtual networks button in all services](images/azs-public-browser-services-virtual-networks.png)
+
+8. Select your virtual network from the list.
+
+9. In the blade for your virtual network you can; view and monitor the virtual network, change its settings and perform diagnostics and troubleshooting.
```

Comment:
No semicolon needed here:
In the blade for your virtual network, you can view and monitor the virtual network, change its settings and perform diagnostics and troubleshooting.